### PR TITLE
Update CODEOWNERS for top level imports.

### PR DESCRIPTION
**Pull request by aaazzam**
This PR makes @chrisguidry and I CODEOWNERS for Prefect's __init__.py and main.py

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -23,5 +23,9 @@ mkdocs.insiders.yml                  @PrefectHQ/docs
 /src/prefect/server/events            @chrisguidry
 /tests/events                         @chrisguidry
 
+# imports
+/src/prefect/__init__.py              @aaazzam @chrisguidry
```
Comment (zzstoatzz) [inline .github/CODEOWNERS:27]: why not the open source team here?

**Review by chrisguidry — APPROVED** [review]

**Review by zzstoatzz — CHANGES_REQUESTED** [review]
linting needs to be run on `main.py` it seems?

**Comment by zhen0** [comment]
+1 to Nate's question - can you add more detail about the need for a change in code owners here?

**Comment by zhen0** [comment]
> @zhen0 see my response to Nate's comments

Thanks for the explanation.

**Review by zzstoatzz — APPROVED** [review]

**Comment by zzstoatzz** [comment]
i dont think it ever makes sense to merge with static analysis failing, but i wont die on that hill here

**Review by jlowin — APPROVED** [review]

**Review by cicdw — APPROVED** [review]
I think this is a clever way to handle managing our bloated imports and the latency they cause - godspeed

**Pull request merged**